src/autocapture.ts

```diff
@@ -255,7 +256,7 @@ const autocapture = {
             const props = _extend(
                 this._getDefaultProperties(e.type),
                 {
-                    $elements: elementsJson,
+                    $elements_chain: getElementsChainString(elementsJson),
```

Comment:
I think we should gradually roll this out to team 2 first for safety, see how capture-rs rollouts are happening